torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -460,6 +478,8 @@ void replaceBlockWithFallbackGraph(Block* b) {
     b->replaceOutput(i, fallback->output(i));
   }
 
+  ProfilingRecord::removeProfilingNodes(graph->block());
```

Comment:
yeah, it's a bit subtle; we need this because `SpecializeAutogradZero` will create a fallback graph with `prim::profile` nodes. We remove those nodes with this function in `replaceBlockWithFallbackGraph`